filemanager/__init__.py

```diff
@@ -106,7 +106,7 @@ def handle_form(self, form, files):
             and not re.match(r'[\w\d_ -]+', name).group(0) == name
         )
         if invalid_folder_name:
-            messages.append("Invalid folder name : " + name)
+            messages.append('{0}{1}'.format("Invalid folder name : ", name))
```

Comment:
`'Invalid folder name : {0}'.format(name)` would be much more appropriate. As a general rule, use the existing string literals as part of the formatting specifier.

Reply:
makes sense. thanks for pointing this out. I figured I would only do 1-2 commits as it was my first attempt.